sound/soc/sof/pcm.c

```diff
@@ -157,7 +157,7 @@ static int sof_pcm_hw_params(struct snd_pcm_substream *substream,
 	pcm.params.buffer_fmt = SOF_IPC_BUFFER_INTERLEAVED;
 	pcm.params.rate = params_rate(params);
 	pcm.params.channels = params_channels(params);
-	pcm.params.host_period_bytes = params_period_bytes(params);
+	pcm.params.host_period_bytes = params_period_size(params);
```

Comment:
so now this is storing frame information in a _bytes field? This is confusing and likely wrong.

Reply:
@keyonjie I remember you have fixed a issue with d6d661c7521a72c2bce6f2ced11b812f449f8e43 in #768
@mrajwa The naming is very confusing here, please give more detail why we should change into something looks wrong.
If really need to do like this, please give inline comments or just change the variable name.

Reply:
@mrajwa no doubt we should use params_period_bytes(), please correct it.

Reply:
Yes, you are right, this commit has been dropped. It worked for our recent tests since in FW I expected period size in frames. But in a longer run it is better to keep this information in bytes.